core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -6252,6 +6286,32 @@ private void checkSemiJoinRuleOnAntiJoin(RelOptRule rule) {
         .check();
   }
 
+  @Test void testExpandProjectInWithTwoCorrelatedSubQueries() {
```

Comment:
These two test cases for expand project in is not affected by this issue, I add these to improve test converage.

Reply:
Thanks for the clarification, I think we can mention this in the commit message when we merging.